doc/sphinx/cp/inelasticity/AsaroInelasticity.rst

```diff
@@ -0,0 +1,45 @@
+AsaroInelasticity
+=================
+
+Overview
+--------
+
+This class implements the standard crystal plasticity plastic deformation
+rate presented in [A1983]_.
+The plastic deformation rate is the sum of simple shear deformations on 
+each individual slip direction and plane:
+
+.. math::
+   \mathbf{L}^p = \sum_{g=1}^{n_{group}}\sum_{i=1}^{n_{slip}}\dot{\gamma}_{g,i}\left(\mathbf{d}_{g,i}\otimes\mathbf{n}_{g,i}\right)
+
+where :math:`\mathbf{d}_{g,i}` are the slip directions and :math:`\mathbf{n}_{g,i}` are
```

Comment:
typo
`\mathbf{d}_{g,i}` and :math:`\mathbf{n}_{g,i}` are are the slip directions

Reply:
Fixed.